<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="swMenu">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>151</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>Matemática</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>151</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 170, 0);</string>
      </property>
      <property name="text">
       <string>Español</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbFondo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>651</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Imagenes.qrc">:/Imagenes/fondomenu.jpg</pixmap>
      </property>
     </widget>
     <zorder>lbFondo</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
    </widget>
    <widget class="QWidget" name="swMatematicas">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>651</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Imagenes.qrc">:/Imagenes/fd1.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>600</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Ingrese su respuesta:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leRespuesta">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>600</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbCorrecto">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>600</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>600</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Aciertos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>620</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fallos</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbAciertos">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>600</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbFallos">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>620</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="swEspanol">
     <widget class="QLabel" name="ayudaLabel">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>171</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="imgLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>581</width>
        <height>451</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbInstrucciones">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>751</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Seleccione la letra correcta para el espacio en blanco.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
